src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class Intake extends SubsystemBase {
+    Intake INSTANCE = null;
+    private IntakeIO io = null;
+    private final IntakeInputsAutoLogged inputs = io.inputs;
+
+    public Intake(IntakeIO io) {
+        this.io = io;
+    }
+
+    public Intake getINSTANCE() {
+        return INSTANCE;
+    }
+
+    public void Initialize(IntakeIO io) {
+        new Intake(io);
+    }
+
+    public Command setAngle(MutableMeasure angle) {
+        return new InstantCommand(() -> io.setAngle(angle));
+    }
+
+    public Command setAngle(IntakePose intakePose) {
+        if (intakePose == IntakePose.UP) {
```

Comment:
It'd be better if the `IntakePose` enum will hold the corresponding measure values.

Reply:
possible

Reply:
done.

Reply:
a2668c70dab52d8f6b7caac96cebb4f97cf1a4ae